app/generic/verifier/verifier.py

```diff
@@ -239,9 +238,12 @@ async def accept_proof_request(
     try:
         prover = __get_verifier_by_version(presentation.proof_id)
 
-        await assert_valid_prover(
-            aries_controller=aries_controller, prover=prover, presentation=presentation
-        )
+        if not presentation.connectionless:
```

Comment:
We can check whether the proof is connectionless by first retrieving it from the ACA-Py instance. This prevents the user needing to provide info that is already availalbe to use and also removes a way for inconsistent state: the user sets connectionless: false, but the exchange is actually connectionless

Reply:
ah alright nice didnt know that - will implement that then